src/avram/query_builder.cr

```diff
@@ -196,7 +212,13 @@ class Avram::QueryBuilder
     @limit || @offset
   end
 
-  def select(selection : Array(Symbol))
+  def selects
+    @selections
+      .split(", ")
+      .map { |column| column.split(".").last }
+  end
+
+  def select(selection : Array(Symbol | String))
```

Comment:
I didn't like this change, but Crystal has no way to turn a String back in to a Symbol, so we have to be able to pass in `Array(String)`.